capstone-app/src/app/modules/photos/services/mock-photo-fetcher.service.ts

```diff
@@ -7,7 +7,7 @@ import { Observable, of } from 'rxjs';
 })
 export class MockPhotoFetcher {
 
-  getPhotos(): Observable<any> {
+  getPhotos(query: string, limit: number): Observable<any> {
```

Comment:
These arguments are not used, please remove.

Reply:
Or add a TODO comment if they are not being utilized yet.

Reply:
I need these arguments here as the constructor for the real getPhotos service uses these arguments